### ISSUE TITLE
Tree-view and git-diff not updated after committing changes

### ISSUE THREAD

**Issue by shadow-light** [atom/github#1700]
### Prerequisites

* Have you tried to reproduce the problem in [Safe Mode][safe-mode]? Yes
* Have you followed all applicable steps in the [debugging guide][debugging-guide]? Yes
* Have you checked the FAQs on the [message board][message-board] for common solutions? Yes
* Have you checked that your issue [isn't already filed][already-filed]? Yes

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

tree-view and git-diff still showing changes after committing those changes.

### Steps to Reproduce

1. Change some files (shown as changed in tree-view and git-diff)
2. Commit the changes using github package

**Expected behavior:**

Changes no longer shown in tree-view and git-diff

**Actual behavior:**

Changes still shown

Changes correctly __no longer__ shown when committing via terminal. Atom detects the commit and updates appropriately.

**Reproduces how often:**

100%

### Versions

```
Atom    : 1.30.0
Electron: 2.0.5
Chrome  : 61.0.3163.100
Node    : 8.9.3

apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.30.0
python 2.7.12
git 2.7.4

Ubuntu 16.04
```

### Additional Information

https://github.com/atom/tree-view/issues/1240 and https://github.com/atom/atom/issues/17994 may be related but there is no problem for me when committing via terminal, just when committing via Atom.

**Comment by rsese** [comment]
Thanks for the report!  Just to confirm from your comments in those other issues, things clear up for you in the tree view and the line diffs if you click out of and back into Atom?

**Comment by shadow-light** [comment]
Yes, closing the changed file's tab and reopening it usually fixes both tree-view and git-diff for that single file. I have to do it individually for each changed file though.

git-diff will also be updated if I make any change to the file.

So I'm usually closing and reopening all the changes files to update their statuses. Any ideas on a quick fix I can add to my init script in the meantime? Something to trigger a refresh of statuses after a commit?

thanks

**Comment by akapps** [comment]
I have the exact same issue here with Atom `1-34-0` on Linux Mint `18.3` (based on `Ubuntu 16.04`).

Interestingly, I found out that losing the focus (such as switching to Firefox for instance) then giving it back to Atom again results in both views being refreshed and up-to-date. That could explain why the problem doesn't show up when committing from an external terminal BTW...  
This makes an acceptable workaround for me, but it would be great if the problem could be fixed in Atom directly.

**Comment by hefayed01** [comment]
Any update on this issue , i found if i hit save (command-s) on mac also the issue go away
